src/loader/LoaderAPI.ts

```diff
@@ -293,9 +293,14 @@ function $getRef(name: string, o: RefOpts): IReference {
         wildcard = validPath;
     }
     if (!file && !wildcard) {
-        // try folder index.js
+        // try index.js
         validPath = $pathJoin(filePath, "/", "index.js");
         file = pkg.f[validPath];
+
+        if (!file && pkg.s && pkg.s.entry) {
```

Comment:
what's that? Why did you modify the loader?

Reply:
If the path is fully relative it will land here. See:
https://github.com/calebboyd/fuse-test 

If its not the index it must be an entry file. Specifically, a fully relative name.

`'..'` or `'../../'`. As a guard an `isRelative` check could be made..

Thoughts?

Reply:
But what if there a package.json with a different entry? that's why we had that `alias` thingy in PathMaster, that overrides it

Reply:
If you have an idea of what `alias` needs to be set to, any tips would be helpful.

Reply:
From what I can tell providing an alias does nothing -- The property is set but then it is never  used again...
Is that correct?
https://github.com/fuse-box/fuse-box/blob/813bc793c7d9de764a5f9aac64889233426f2c0a/src/core/PathMaster.ts#L153-L155